server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/DeleteAction.java

```diff
@@ -91,34 +89,36 @@ public void handle(Request request, Response response) throws Exception {
     }
   }
 
-  private void checkNotTryingToDeleteDefaultGroup(DbSession dbSession, long groupId) {
+  private void checkNotTryingToDeleteDefaultGroup(DbSession dbSession, GroupId group) {
+    // TODO support organizations
     String defaultGroupName = settings.getString(CoreProperties.CORE_DEFAULT_GROUP);
     GroupDto defaultGroup = dbClient.groupDao().selectOrFailByName(dbSession, defaultGroupName);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "selectOrFailByName"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)